common/models/config-file.js

```diff
@@ -248,10 +246,11 @@ module.exports = function(ConfigFile) {
 
       cb(null, result);
     });
-  }
+  };
 
   function entityBelongsToFacet(name, definition) {
-    return definition && definition.properties && definition.properties.facetName;
+    return definition && definition.properties &&
+      definition.properties.facetName;
```

Comment:
> Please split into three lines, see https://github.com/strongloop-internal/loopback-knowledge-base/blob/4772f51fcc68c7c1a83157b31eac5379e509d238/style-guide.md#indentation-of-multi-line-expressions-in-return (the code examples don't explicitly say it, would you mind submitting a PR to fix that?)

Hmm, on second thought, that may not be the best idea. Let's either keep what you have now (and ignore my comment entirely), or refactor it as shown in the second snippet.

```js
var properties = definition && definition.properties;
return properties && properties.facetName;
```